lib/dry/component/container.rb

```diff
@@ -34,6 +34,10 @@ def self.configure(env = config.env, &block)
         self
       end
 
+      def self.Loader(key)
```

Comment:
I don't think I follow what this is for here :( Would you mind clarifying the intention? I'm not the one merging this, just reading out of curiosity to learn what you did :)